tox_encryptsave/src/lib.rs

```diff
@@ -354,7 +335,6 @@ pub enum KeyDerivationError {
     #[fail(display = "Provided passphrase is empty")]
```

Comment:
Do we still get all errors of KeyDerivationError?

Reply:
Yes, we have both of them.